src/main/java/uk/gov/hmcts/cmc/claimstore/events/claim/PostClaimOrchestrationHandler.java

```diff
@@ -17,37 +17,21 @@
 import uk.gov.hmcts.cmc.claimstore.stereotypes.LogExecutionTime;
 import uk.gov.hmcts.cmc.domain.models.Claim;
 import uk.gov.hmcts.cmc.domain.models.ClaimState;
-import uk.gov.hmcts.cmc.domain.models.ClaimSubmissionOperationIndicators;
-import uk.gov.hmcts.cmc.domain.models.response.YesNoOption;
 
 import java.util.function.Function;
-import java.util.function.Predicate;
 import java.util.function.Supplier;
-import java.util.stream.Stream;
+
+import static uk.gov.hmcts.cmc.domain.models.response.YesNoOption.NO;
 
 @Async("threadPoolTaskExecutor")
 @Service
 @ConditionalOnProperty(prefix = "feature_toggles", name = "async_event_operations_enabled", havingValue = "true")
 public class PostClaimOrchestrationHandler {
     private static final Logger logger = LoggerFactory.getLogger(PostClaimOrchestrationHandler.class);
+
     private final DocumentOrchestrationService documentOrchestrationService;
     private final ClaimService claimService;
 
-    private final Predicate<ClaimSubmissionOperationIndicators> isPinOperationSuccess = indicators ->
-        Stream.of(indicators.getBulkPrint(), indicators.getStaffNotification(),
-            indicators.getDefendantNotification())
-            .anyMatch(ind -> ind.equals(YesNoOption.NO));
-    private final Predicate<ClaimSubmissionOperationIndicators> isUploadSealedClaimSuccess = indicators ->
-        indicators.getSealedClaimUpload().equals(YesNoOption.NO);
-    private final Predicate<ClaimSubmissionOperationIndicators> isUploadClaimReceiptSuccess = indicators ->
-        indicators.getClaimIssueReceiptUpload().equals(YesNoOption.NO);
-    private final Predicate<ClaimSubmissionOperationIndicators> isRpaOperationSuccess = indicators ->
-        indicators.getRpa().equals(YesNoOption.NO);
-    private final Predicate<ClaimSubmissionOperationIndicators> isNotifyStaffSuccess = indicators ->
-        indicators.getStaffNotification().equals(YesNoOption.NO);
-    private final Predicate<ClaimSubmissionOperationIndicators> isNotifyCitizenSuccess = indicators ->
-        indicators.getClaimantNotification().equals(YesNoOption.NO);
-
```

Comment:
NOTE: Clean up all these predicate functions as these were never used as the assignment target for a lambda expression or method reference. Rather these were misused to muddle the code. The names were also misleading for these.